gensim/models/_fasttext_bin.py

```diff
@@ -168,6 +185,7 @@ def _load_vocab(fin, new_format, encoding='utf-8'):
         The loaded vocabulary.  Keys are words, values are counts.
         The vocabulary size.
         The number of words.
+        The numnber of tokens.
```

Comment:
```suggestion
        The number of tokens.
```

Reply:
Corrected.

As far as motivation goes. I did not have very specific use case in mind.
Improving interoperability of gensim with original fastText seemed like a nice feature.

One of the loose ideas I had, was to try fine tuning gensim model using fastText. However FB, still does not support incremental learning, 

https://github.com/facebookresearch/fastText/pull/423

so I believe this is not yet possible.